### ISSUE TITLE
Spectra Editor does not work with 195Pt NMR

### ISSUE THREAD

**Issue by schatzsc** [ComPlat/chemotion_ELN#1685]
In Chemotion 1.8.0 the Spectra Editor works with 1H and 13C NMR now but seriously messes up the import of 195Pt NMR:

This is what the spectrum looks like when opened with Bruker TopSpin 4.1.1

![195 Pt NMR TopSpin](https://github.com/ComPlat/chemotion_ELN/assets/88166560/672aa32e-8924-4788-9d8d-9c4406447f52)

This is what the "Analyses" preview window looks like after the upload of the same ZIP file to our Chemotion installation:

![Chemotion SpectralViewer1](https://github.com/ComPlat/chemotion_ELN/assets/88166560/d299bb64-827b-4d91-8397-39b435fecc5e)

And this is what the Spectra Viewer presents to me when I open it with the light blue button:

![Chemotion SpectralViewer2](https://github.com/ComPlat/chemotion_ELN/assets/88166560/56dfbebc-9fc0-499c-970b-977dc659b578)

The strange thing is, this latter spectrum looks identical to the 13C NMR that has been uploaded just before, but correctly displays the ZIP file name as "33" while the 13C NMR is ZIP file "32".

It seems like Chemotion somehow "halluzinates along" the 13C NMR when somehow is not able to process the 195Pt NMR - possibly because the measurement range is -2350 to -2850 ppm which is a totally normal range for 195Pt

195Pt NMR file here:

[33.zip](https://github.com/ComPlat/chemotion_ELN/files/13922381/33.zip)

Chemotion should be "agnostic" to the nucleus and measurement range of any uploaded spectrum and provide a "generic NMR spectrum" which fits all nuclei automatically. After all, they are generally the same, independent of nucleus, with intensity on the y-axis and chemical shift on the x-axis

Bug level: very serious

**Comment by hamed-musallam** [comment]
Hello @schatzsc 

Are you sure from the uploaded file as i can see from the row files, it is 13C and not 195Pt

![Screenshot 2024-02-15 at 11 32 41](https://github.com/ComPlat/chemotion_ELN/assets/35760236/bf9c8f63-24ab-4bc2-9422-521f07235f82)

**Comment by schatzsc** [comment]
Yes, you are right - I fell victim to the trap that normally the different sub-folders are assigned with increasing numbers corresponding to increasing atomic number which would have made ../32 the 13C and ../33 the 195Pt but for reasons unknown to me it is reversed here and ../32 is actually the 195Pt NMR which is attached now:

[32.zip](https://github.com/ComPlat/chemotion_ELN/files/14296783/32.zip)

**Comment by schatzsc** [comment]
@hamed-musallam I just looked a bit deeper and the strange "step function peak" shown above is from the ../pdata/701 sub-folder in the 32.zip archive file, while the actual spectrum is in subfolder ../pdata/1

However, the 13C NMR attached above as 33.zip also has two additional subfolders ../pdata/700 and ../pdata/701 and still displays correctly

**Comment by hamed-musallam** [comment]
i tested the 32.zip file with NMRium which is supposed to be supported with Chemotion it works well 
![image](https://github.com/ComPlat/chemotion_ELN/assets/35760236/e327a5b2-f67f-42db-9a1d-a3ba73db6f00)

**Comment by schatzsc** [comment]
Yes, that's the intended behaviour - but NMRium is not included in our current Chemotion instance and it makes it even more weird that the internal spectral editor does not display it the same way?!?

As it is the same x/y data of intensity over chemical shift and the other parameters and metadata fields included in the original Bruker file have the same structure as 1H and 13C it must be some small "hard-coded" thing in the spectral editor that prevents it from displayed correctly.

**Comment by CS76** [comment]
@schatzsc Would you like to enable NMRium on your instance?? 

With regards to the chemspectra @baolanlequang will be able to look into this.

**Comment by schatzsc** [comment]
@CS76 Our colleagues from the computer center first need to look into whether we will set up a local instance or use an external one, which will likely take a couple of days to discuss. Will come back to you in case there are problems.

**Comment by nicolejung** [comment]
I used the 32.zip file and got 4 spectra which can be selected according to the subfolders in the original file. 1 full spectrum, 1 full spectrum processed, 2 spectra processed and delivered with limited range.
Depending on the selection of the file, ChemSpectra shows different results (as defined in those files). Two of the spectra contain the full measurement range as expected - the others not. ChemSpectra only displays them as provided (as far as I can see). 
The last edited spectrum is usually selected as a preview (-> the first image in this github discussion is just the automatically generated preview where chemspectra selects one of the provided files). The view as given in the ChemSpectra editor (with the selection of all available files) is given here:
![Screenshot (1338)](https://github.com/ComPlat/chemotion_ELN/assets/7559189/3021d97e-c691-4b1a-94a3-71e1375ea22a)
One may discuss why the processing is not as good as in NMRium (the software to be used for special NMR topics) but actually I don't see the point of spectral range.

**Comment by schatzsc** [comment]
Inspired by @PiTrem recent comments on [issue 1826](https://github.com/ComPlat/chemotion_ELN/issues/1826) I exported the 195Pt NMR [32.zip](https://github.com/ComPlat/chemotion_ELN/files/14296783/32.zip) from TopSpin 4.1.1 as a *.dx file (included below as ZIP archive since *.dx upload not allowed) and uploaded it to Chemotion 1.8.0 where the spectrum editor surprisingly gives a result similar to TopSpin and also the one shown above for NMRium by @hamed-musallam which is the intended behaviour

I am still puzzled what goes wrong when the internal Spectra Editor reads the FID instead?!?

**Furthermore, the x-axis is wrong, as it should run from -2350 to -2850 ppm**

![195Pt NMR export dx](https://github.com/ComPlat/chemotion_ELN/assets/88166560/ad2c5119-baae-4eb4-a846-30145fbc321f)

[nmr.zip](https://github.com/ComPlat/chemotion_ELN/files/14749842/nmr.zip)

**Comment by schatzsc** [comment]
This is the correct display in TopSpin:

![Uploading Screenshot 2024-03-27 162331.jpg…]()

**Comment by schatzsc** [comment]
@PiTrem @nicolejung The whole issue gets even more confusing since the online version of NMRium at https://www.nmrium.org/nmrium displays the spectrum correctly:

https://github.com/cheminfo/nmrium/issues/2999

![Screenshot 2024-03-28 at 18-45-44 NMRium](https://github.com/ComPlat/chemotion_ELN/assets/88166560/dd6d030b-8065-4a3d-b50a-5f59978118b0)

Are there differences in the versions of NMRium stand-alone and embedded in Chemotion?!?

**Comment by CS76** [comment]
> Are there differences in the versions of NMRium stand-alone and embedded in Chemotion?!?

@schatzsc Yes, there are; https://www.nmrium.org/nmrium is the latest NMRium, which might have fixes to resolve this issue.

@baolanlequang, Can we check if there are any major updates to the data model from the current Chemotion-compatible NMRium RW version to the latest? If not we could upgrade it and check if the issues are resolved.

**Comment by schatzsc** [comment]
@baolanlequang @CS76 @nicolejung 

Could you please urgently consult with the NMRium team about their latest version and its integration into Chemotion, as some of the problems mentioned above are solved in the online test version of NMRium but don't work with the one that is provided with Chemotion?

**Comment by schatzsc** [comment]
@baolanlequang @CS76 @PiTrem @nicolejung

Following our discussion in the Zoom chat this morning I would like to point out again that NMRium seems to have generally **solved the issues with the different non-standard NMR nuclei** (beyond 1H, 13C, and a few others) **by adding the gyromagnetic ratio for all NMR-active nuclei** to their internal data structure.

The discussion over at the NMRium GitHub is in [issue #2999](https://github.com/cheminfo/nmrium/issues/2999)

Gyromagnetic ratio of some common NMR nuclei are reported in [Pure Appl. Chem. 2001, 73, 1795-1818](https://dx.doi.org/10.1351/pac200173111795) however relative to the ones used in NMRium they are off by a factor of 10 possibly due to differences in units (10^7 rad s^-1 T^-1 vs. ???)

![Screenshot 2024-05-21 180122](https://github.com/ComPlat/chemotion_ELN/assets/88166560/b7382fbc-0a55-470c-8dcc-0f930f919d5e)
